--- a/src/d_netfil.c	2022-11-04 17:11:36.737717055 -0300
+++ b/src/d_netfil.c	2022-11-04 17:13:44.473715289 -0300
@@ -343,15 +343,6 @@
 		}
 	}
 
-	I_GetDiskFreeSpace(&availablefreespace);
-	if (totalfreespaceneeded > availablefreespace)
-	{
-		CONS_Printf("Direct download -\n"
-			" To play on this server you must download %s KB,\n"
-			" but you have only %s KB free space on this drive\n",
-			sizeu1((size_t)(totalfreespaceneeded>>10)), sizeu2((size_t)(availablefreespace>>10)));
-		return false;
-	}
 
 #ifdef MORELEGACYDOWNLOADER
 tryagain:
