--- a/src/d_netfil.c	2022-10-13 15:06:54.375807703 -0300
+++ b/src/d_netfil.c	2022-10-13 16:21:25.557745874 -0300
@@ -338,11 +338,6 @@
 			fileneeded[i].status = FS_REQUESTED;
 		}
 	WRITEUINT8(p, 0xFF);
-	I_GetDiskFreeSpace(&availablefreespace);
-	if (totalfreespaceneeded > availablefreespace)
-		I_Error("To play on this server you must download %s KB,\n"
-			"but you have only %s KB free space on this drive\n",
-			sizeu1((size_t)(totalfreespaceneeded>>10)), sizeu2((size_t)(availablefreespace>>10)));
 
 	// prepare to download
 	I_mkdir(downloaddir, 0755);
